pyccel/codegen/printing/ccode.py

```diff
@@ -55,6 +56,105 @@
     "ceiling": "ceil",
 }
 
+# dictionary mapping numpy function to (argument_conditions, C_function).
+# Used in CCodePrinter._print_NumpyUfuncBase(self, expr)
+numpy_ufunc_to_c = {
+    'NumpyAbs'  : 'fabs',
+    'NumpyMin'  : 'minval',
+    'NumpyMax'  : 'maxval',
+    'NumpyFloor': 'floor',  # TODO: might require special treatment with casting
+    # ---
+    'NumpyExp' : 'exp',
+    'NumpyLog' : 'log',
+    'NumpySqrt': 'sqrt',  # sqrt is printed using _Print_NumpySqrt
+    # ---
+    'NumpySin'    : 'sin',
+    'NumpyCos'    : 'cos',
+    'NumpyTan'    : 'tan',
+    'NumpyArcsin' : 'asin',
+    'NumpyArccos' : 'acos',
+    'NumpyArctan' : 'atan',
+    'NumpyArctan2': 'atan2',
+    'NumpySinh'   : 'sinh',
+    'NumpyCosh'   : 'cosh',
+    'NumpyTanh'   : 'tanh',
+    'NumpyArcsinh': 'asinh',
+    'NumpyArccosh': 'acosh',
+    'NumpyArctanh': 'atanh',
+}
+
+# dictionary mapping Math function to (argument_conditions, C_function).
+# Used in CCodePrinter._print_MathFunctionBase(self, expr)
+# Math function ref https://docs.python.org/3/library/math.html
+math_function_to_c = {
+    # ---------- Number-theoretic and representation functions ------------
+    'MathCeil'     : 'ceil',
+    # 'MathComb'   : 'com' # TODO
+    'MathCopysign': 'copysign',
+    'MathFabs'   : 'fabs',
+    # 'MathFactorial': '???', # TODO
+    'MathFloor'    : 'floor',
+    # 'MathFmod'   : '???',  # TODO
+    # 'MathRexp'   : '???'   TODO requires two output
+    # 'MathFsum'   : '???',  # TODO
+    # 'MathGcd'   : '???',  # TODO
+    # 'MathIsclose' : '???',  # TODO
+    'MathIsfinite': 'isfinite', # int isfinite(real-floating x);
+    'MathIsinf'   : 'isinf', # int isinf(real-floating x);
+    'MathIsnan'   : 'isnan', # int isnan(real-floating x);
+    # 'MathIsqrt'  : '???' TODO
```

Comment:
What is this function? Is it a typo?

Reply:
are you talking about the comments?
I was going to work offline, I put them in comment to remember the prototype of function

Reply:
I'm talking about `MathIsqrt`. I assume anything commented is something that is not implemented yet, but I don't see a `math.isqrt` in python

Reply:
it exists in newer versions of python.
math.isqrt(n) :
Return the integer square root of the nonnegative integer n. This is the floor of the exact square root of n, or equivalently the greatest integer a such that a² ≤ n.
for more info check the [python doc](https://docs.python.org/3/library/math.html#math.isqrt)